app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -104,14 +103,14 @@ function applyEditCallBack() {
     toggleButtons(false);
     initializeCheckboxSorting();
 
+    TinyMCE.init();
```

Comment:
Can't we move this out globally as we did with `quill.js`? Or is it not possible?